src/lightning_app/core/app.py

```diff
@@ -162,6 +165,12 @@ def __init__(
         # this should happen once for all apps before the ui server starts running.
         frontend.update_index_file(FRONTEND_DIR, info=info, root_path=root_path)
 
+        if debugger_is_active() and not bool(int(os.getenv("LIGHTNING_DISPATCHED", "0"))):
```

Comment:
see #15388

Reply:
Not sure to follow you there. There is a `LIGHTNING` prefix. What would you prefer ?